src/main/java/com/internship/socialnetwork/controller/UserController.java

```diff
@@ -15,37 +18,78 @@
 
 import java.util.List;
 
+import static org.springframework.http.HttpStatus.CREATED;
+
 @RestController
 @RequiredArgsConstructor
 @RequestMapping("/api/v1/users")
 public class UserController {
 
     private final UserService userService;
 
+    private final FriendshipService friendshipService;
+
     @GetMapping
-    public ResponseEntity<List<UserDto>> getAll() {
+    public ResponseEntity<List<UserDto>> getAllUsers() {
         return ResponseEntity.ok(userService.getAllUsers());
     }
 
     @GetMapping("/{id}")
-    public ResponseEntity<UserDto> get(@PathVariable Long id) {
+    public ResponseEntity<UserDto> getUser(@PathVariable Long id) {
         return ResponseEntity.ok(userService.getUser(id));
     }
 
+    @GetMapping("/{id}/friends")
+    public ResponseEntity<List<FriendshipDto>> getFriends(@PathVariable Long id) {
+        return ResponseEntity.ok(friendshipService.getFriends(id));
+    }
+
+    @GetMapping("/{id}/friends/requests")
```

Comment:
should be either /{id}/requests or /{id}/friend-requests. This is not a RESTful name